dpgen/data/gen.py

```diff
@@ -805,35 +808,7 @@ def pert_scaled(jdata):
                 else:
                     shutil.copy2(pos_in, pos_out)
                 os.remove(pos_in)
-            kk = -1
```

Comment:
So long as these codes are deleted, where are the new codes corresponding to them?

Reply:
They are no longer needed. Please let the others know the new code work or not?